python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -650,3 +651,48 @@ def post_tune(self, task_scheduler, task_id):
                 % (time.time() - task_scheduler.tic, total_latency_str, task_scheduler.ct)
```

Comment:
As I said. Change %-formating to fstring formatting, e.g. %.0f to {time.time():.0f}